src/helm/benchmark/static/schema_classic.yaml

```diff
@@ -2232,6 +2232,107 @@ run_groups:
       when: n/a
       language: synthetic
 
+## Instruction following scenarios
```

Comment:
Why are these not in `schema_instruction_following.yaml`?

Reply:
Reverted `schema_classic.yaml`.